study.md

```diff
@@ -50,7 +50,7 @@ Does `proxima_centauri.print_mass` run? Why or why not? If so, then what is the
 output? If not, then how can it be fixed?
 
 ```md
-<!-- your answer here -->
+No, the named variable in the print_solar needs to be changed in two places (line 23 and 25) to be print_kg.
```

Comment:
Neither `print_solar` nor `print_kg` are identifiers.

This method doesn't work because it attempts to access a local variable called mass_kg that doesn't exist.  One way to fix it is to set mass_kg to an instance variable in the constructor.